zenguard/zenguard.py

```diff
@@ -87,6 +80,22 @@ def detect(self, detectors: list[Detector], prompt: str):
             return {"error": str(e)}
 
         return response.json()
+    
+    def detect_and_send_prompt(self, detectors: list[Detector], prompt: str):
+        try:
+            detected_response = self.detect(detectors, prompt)
+            if "error" in detected_response:
+                return detected_response
+            llm_response = self._llm_client.chat.completions.create(
+                model="gpt-3.5-turbo",
```

Comment:
stuff like this(model, system message, temperature) should be decided by the caller and not by us.